wandb/filesync/dir_watcher.py

```diff
@@ -98,10 +98,8 @@ class PolicyLive(FileEventHandler):
     """This policy will upload files every RATE_LIMIT_SECONDS as it
     changes throttling as the size increases"""
 
-    TEN_MB = 10000000
-    HUNDRED_MB = 100000000
-    ONE_GB = 1000000000
     RATE_LIMIT_SECONDS = 15
+    unit_dict = dict(util.POW_10_BYTES)
```

Comment:
😄

Reply:
why 😄? haha

Reply:
it's just joy :) i like it without the weird "TEN_MB" constants

Reply:
I actually think the constants are better than arbitrary dicts as they can be statically checked.

Reply:
the dict is derived from a tuple of constants (so it is not arbitrary)